macros/adapters/default/page_views/snowplow_web_events.sql

```diff
@@ -15,7 +15,7 @@
 {{
     config(
         materialized='incremental',
-        sort='page_view_id',
+        sort=['collector_tstamp', 'page_view_id', 'domain_sessionid', 'domain_userid', 'dvce_created_tstamp'],
```

Comment:
I think a compound sort key on `collector_tstamp` + `page_view_id` makes sense. Could you make this change in a separate PR? I understand it's motivated by `snowplow_page_views` performance, but I think it's outside the scope of tweaking the model SQL of `snowplow_page_views` in particular.